grammar.js

```diff
@@ -69,7 +70,37 @@ module.exports = grammar({
   rules: {
     translation_unit: $ => repeat($._top_level_item),
 
+    // Top level items are block items with the exception of the expression statement
     _top_level_item: $ => choice(
```

Comment:
One issue with this is that if one day one add a new kind of statement, then he will need to add it also there, and it's easy to forget. If instead we factorize things with
block_item_no_expr_statement,  and then have block_item: choice(block_item_no_expr_statement, expr_statement),
then we would need to modify only one place.